Doc/library/pty.rst

```diff
@@ -43,11 +43,32 @@ The :mod:`pty` module defines the following functions:
 
    Spawn a process, and connect its controlling terminal with the current
    process's standard io. This is often used to baffle programs which insist on
-   reading from the controlling terminal.
+   reading from the controlling terminal. It is expected that the process
+   spawned behind the pty will eventually terminate, and when it does *spawn*
+   will return.
+
+   The functions *master_read* and *stdin_read* are passed a file descriptor
+   which they should read from, and they should always return a byte string. In
+   order to force spawn to return before the child process exits an *OsError*
+   should be thrown.
+
+   The default implementation for both functions will read and return up to 1024
+   bytes each time the function is called. *Master_read* is passed the
```

Comment:
``Master-read`` shouldn't be capitalized since it's an argument.  You can start the sentence with the lower case or start the sentence with the literal ``The argument *master_read*``.